README.md

```diff
@@ -7,367 +7,600 @@
+- a number of [built-in pretty output formatters](#formatting)
+- easy [configuration](#list-of-options)
+- the ability to [extend](#extending)
+- showing colored diff when using the [`t.equal()`](#tequalactual-any-expected-any-message-string-since-v102) and [`t.deepEqual()`](#tdeepequalactual-any-expected-any-message-string-since-v102) assertion operators
+- detailed stack traces for `async` functions
+- not needing to specify the number of assertions with [`t.plan()`](https://github.com/substack/tape#tplann)
+- multiple [`test.only`'s](#testonlymessage-string-fn-t-test--void-options-testoptions-since-v100)
+- [smart timeouts](#supertape_timeout) for long running tests 🏃‍♂️
+- more natural assertions: `expected, actual` -> `actual, expected`:
+
+  ```js
+  t.equal(error.message, 'hello world', `expected error.message to be 'hello world'`);
+  ```
 
-Doesn't contain:
+📼 **Supertape** doesn't contain:
 
-- `es3 code` and lot's of [ponyfills](https://github.com/sindresorhus/ponyfill#how-are-ponyfills-better-than-polyfills).
-- aliases, methods list much shorter;
-- `throws`, `doesNotThrows` - use [tryCatch](https://github.com/coderaiser/try-catch), [tryToCatch](https://github.com/coderaiser/try-to-catch) with `equal` instead.
+- assertion aliases, making the available operators far more concise
+- `es3 code` and lot's of [ponyfills](https://github.com/sindresorhus/ponyfill#how-are-ponyfills-better-than-polyfills)
+- `t.throws()`, `t.doesNotThrow()` - use [**tryCatch**](https://github.com/coderaiser/try-catch) or [**tryToCatch**](https://github.com/coderaiser/try-to-catch) with `t.equal()` instead
+
+For a list of all built-in assertions, see [Operators](#operators).
 
 ## Install
 
 ```
 npm i supertape -D
 ```
 
-## Usage
+## Example
+
+```js
+// test.js
+const test = require('supertape');
+
+test('hello: world', (t) => {
+    t.pass('hello, world!');
+    t.end();
+});
+```
+
+Output:
+
+```
+$ npx supertape test.js
+TAP version 13
+
+1..1
+# tests 1
+# pass 1
+
+# ✅ ok
+```
+
+## Usage and Configuration
+
+Tests can be run either through the `supertape` CLI or through `node` (or `ts-node` if using **TypeScript**). [Options](#list-of-options) can be specified via the command line, through environment variables, or in individual test files.
+
+### Command Line
+
+📼 **Supertape** provides a CLI that supports running both `CJS` and `ESM` tests:
 
 ```
-Usage: supertape [options] [path]
-Options
-   -h, --help                  display this help and exit
-   -v, --version               output version information and exit
-   -f, --format                use a specific output format - default: progress-bar/tap on CI
-   -r, --require               require module
-   --no-check-scopes           do not check that messages contains scope: 'scope: message'
-   --no-check-assertions-count do not check that assertion count is no more then 1
-   --no-check-duplicates       do not check messages for duplicates
+$ [npx] supertape "tests/test1.js" [options]
```

Comment:
I would prefer to keep cli output

Reply:
Readded with https://github.com/tommy-mitchell/supertape/commit/2805a22063dab5421937735453c867724891245b